jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListViewImpl.java

```diff
@@ -208,4 +208,10 @@ public void initColumns(final ListTable<ExecutionErrorSummary> extendedPagedTabl
                         true)
         );
     }
-}
\ No newline at end of file
+
+    @Override
+    public Column<ExecutionErrorSummary, String> getSecondSortColumn() {
+        return createTextColumn(COLUMN_ERROR_ID,
```

Comment:
@bxf12315 there is enough space to be in the same line

Reply:
ok